packages/features/bookings/lib/handleNewBooking/getLocationValuesForDb.ts

```diff
@@ -42,9 +42,17 @@ const _getLocationValuesForDb = <
         credentials: firstDynamicGroupMember.credentials,
       });
 
-    firstDynamicGroupMemberDefaultLocationUrl =
-      firstDynamicGroupMemberMetadata?.defaultConferencingApp?.appLink ||
-      firstDynamicGroupMemberDelegationCredentialConferencingAppLocation;
+    const defaultConferencingApp = firstDynamicGroupMemberMetadata?.defaultConferencingApp;
+
+    const hasMemberSetConferencingPreference =
```

Comment:
Important difference here is that now appSlug and appLink both can individually decide if preference is set. Earlier it was just `appLink` which is set for static link conferencing apps but not for dynamic ones like Zoom, Cal Video